library/ix-dev/charts/pihole/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: DNS and Ad-filtering for your network.
 annotations:
   title: Pi-hole
 type: application
-version: 2.0.6
+version: 2.0.7
 apiVersion: v2
 appVersion: 2024.02.0
```

Comment:
Hello again, do you need to change something here? Because in the Pi-Hole Container was a update on TrueNAS scale but it has the same App Version as before internally.
<img width="467" alt="image" src="https://github.com/truenas/charts/assets/96265421/71edaa2c-b722-4616-8ef4-8f49baff2b79">

Reply:
Hello, see this https://github.com/truenas/charts/pull/2288 please